src/services/contract_wrappers/contract_wrapper.js

```diff
@@ -7,6 +7,7 @@ This Source Code Form is subject to the terms of the Mozilla Public License, v.
 This Source Code Form is “Incompatible With Secondary Licenses”, as defined by the Mozilla Public License, v. 2.0.
 */
 
+/** @abstract */
```

Comment:
Could you also add this to `HermesUploadStrategy`?

Reply:
Guess what!
https://github.com/ambrosus/ambrosus-node/pull/204#discussion_r209900548

Reply:
🙈 🙉 🙊